crates/aptos-jwk-consensus/src/jwk_observer.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright © Aptos Foundation
+
+use anyhow::Result;
+use aptos_channels::aptos_channel;
+use aptos_logger::{debug, info};
+use aptos_types::jwks::{jwk::JWK, Issuer};
+use futures::{FutureExt, StreamExt};
+use move_core_types::account_address::AccountAddress;
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+use tokio::{sync::oneshot, task::JoinHandle, time::MissedTickBehavior};
+
+#[derive(Serialize, Deserialize)]
+struct OpenIDConfiguration {
+    issuer: String,
+    jwks_uri: String,
```

Comment:
uri or url?

Reply:
uri by design. that's what's returned [here](https://www.facebook.com/.well-known/openid-configuration/)